frontend/src/components/Comment/Comment.js

```diff
@@ -5,6 +5,7 @@ import EditComment from './EditComment'
 import timeElapsedSincePosted from '../../services/TimeService'
 
 import './SCSS/Comment.scss'
+import LikeButtonComment from "../LikeButtonComment/LikeButtonComment"
```

Comment:
can you put this import between `EditComment` and `timeElapsedSincePosted` to comply to the coding standards, please?